### ISSUE TITLE
Verify current state of /users/invite endpoint implementation

### ISSUE THREAD

**Issue by honzabilek4** [directus/api#661]
Hi @WellingGuzman ,
what is the current state of the `/users/invite` endpoint? Is there any additional work to be done in order to make it work?

I tried to test the endpoint and I've successfully received the invitation email. Though the email still going to need some care. The links there are incorrect and it's a little bit broken in my client (missing logo, too wide because of the link..)

**Comment by WellingGuzman** [comment]
Hey @honzabilek4, can you show us the email you received, and how the broken url looked like?

One thing about Directus and URL, is that at the moment most URL are based on settings primarily the mail URL info. I believe by default all this URL should be guessed based on the request.

See comment here regarding a similar issue (_I suppose_): https://github.com/directus/api/issues/597#issuecomment-440719838.

**Comment by honzabilek4** [comment]
The email looks exactly the same as in #597, so fixing that issue would solve the problem here.

Apart from that, the endpoint is ready to use, correct?

**Comment by WellingGuzman** [comment]
Yes, the endpoint is ready to use. Did the invitation create an user with the status of `invited`? if that happened then the endpoint is ready to use.
